search-service/src/main/java/de/cxp/ocs/util/SearchQueryBuilder.java

```diff
@@ -252,4 +252,7 @@ public String toString() {
 		return searchQueryLink.getRawQuery();
 	}
 
+	public Map<String, InternalResultFilter> getFilters() {
```

Comment:
This exposes internal state of this class and risks of being mutated. Until now it is immutable, which is very import for this class.

Reply:
Ok, I've removed it, thanks.